node/packages/sdk/lib/trace-span.js

```diff
@@ -137,7 +135,8 @@ class TraceSpan {
         const message =
           "Serverless SDK Warning: Following trace spans didn't end before end of " +
           `lambda invocation: ${leftoverSpans.map(({ name }) => name).join(', ')}\n`;
-        this._reportWarning(message, 'SDK_SPAN_NOT_CLOSED', { type: 'USER' });
+        // Require on spot to avoid otherwise difficult to mitigate circular dependency
+        require('..')._reportWarning(message, 'SDK_SPAN_NOT_CLOSED', { type: 'USER' });
```

Comment:
If it can be avoided it's best to avoid it. Still, I think it's better than passing value through multiple layers (in this scenario it actually opened room for bug in context of AWS Lambda package - will patch with next PR)